contracts/src/deployers/aerodrome-lp/AerodromeLpHyperdriveDeployerCoordinator.sol

```diff
@@ -0,0 +1,160 @@
+        ERC20 baseToken = ERC20(_hyperdrive.baseToken());
+        baseToken.safeTransferFrom(_lp, address(this), _contribution);
+        baseToken.forceApprove(address(_hyperdrive), _contribution);
+
+        // This yield source isn't payable, so we should always send 0 value.
+        return 0;
+    }
+
+    /// @notice Convert an amount of vault shares to an amount of base.
+    /// @param _shareAmount The vault shares amount.
+    /// @return The base amount.
+    function convertToBase(uint256 _shareAmount) public pure returns (uint256) {
+        return AerodromeLpConversions.convertToBase(_shareAmount);
+    }
+
+    /// @notice Convert an amount of base to an amount of vault shares.
+    /// @param _baseAmount The base amount.
+    /// @return The vault shares amount.
+    function convertToShares(
+        uint256 _baseAmount
+    ) public pure returns (uint256) {
+        return AerodromeLpConversions.convertToShares(_baseAmount);
+    }
+
+    /// @dev We override the message value check since this integration is
+    ///      not payable.
+    function _checkMessageValue() internal view override {
+        if (msg.value != 0) {
+            revert IHyperdriveDeployerCoordinator.NotPayable();
+        }
+    }
+
+    /// @notice Checks the pool configuration to ensure that it is valid.
+    /// @param _deployConfig The deploy configuration of the Hyperdrive pool.
+    /// @param _extraData The empty extra data.
+    function _checkPoolConfig(
+        IHyperdrive.PoolDeployConfig memory _deployConfig,
+        bytes memory _extraData
+    ) internal view override {
+        // Perform the default checks.
+        super._checkPoolConfig(_deployConfig, _extraData);
+
+        // Ensure that the vault shares token address is properly configured.
+        // Since we don't have a vault for the lp tokens, the address should
+        // be zero.
+        if (address(_deployConfig.vaultSharesToken) != address(0)) {
+            revert IHyperdriveDeployerCoordinator.InvalidVaultSharesToken();
+        }
+
+        // Ensure that the base token address is properly configured.
+        if (address(_deployConfig.baseToken) == address(0)) {
+            revert IHyperdriveDeployerCoordinator.InvalidBaseToken();
+        }
+
+        // Ensure that the minimum share reserves are equal to 1e7. This value
+        // has been tested to prevent arithmetic overflows in the
+        // `_updateLiquidity` function when the share reserves are as high as
+        // 200 million.
+        if (_deployConfig.minimumShareReserves != 1e7) {
+            revert IHyperdriveDeployerCoordinator.InvalidMinimumShareReserves();
+        }
+
+        // Ensure that the minimum transaction amount are equal to 1e7. This
+        // value has been tested to prevent precision issues.
+        if (_deployConfig.minimumTransactionAmount != 1e7) {
+            revert IHyperdriveDeployerCoordinator
+                .InvalidMinimumTransactionAmount();
+        }
```

Comment:
These values are really low. I know that the supply is low, but what's the rationale for these being set so low?

Reply:
it's an 18-decimal token but it takes very low values. the entire pool is worth $150M and it's just 1.23 LP tokens (1.23e18), so 1e7 is like $0.0012 USD

Reply:
Currently set this to 1e8 which is like 1 cent currently.  1e10 would be about a dollar, which would also be reasonable